collections/_programs/hacktoberfest_2020.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: project
+title: "Hacktoberfest"
+image:
+path: /assets/images/buttons/gsoc.png
+thumbnail: /assets/images/buttons/Icon.svg
+link: /programs/hacktoberfest
+permalink: /programs/hacktoberfest
+excerpt: "Layer5 - Hacktoberfest"
+---
+<style>
+  div.gsoc {
+    margin-left: 15px;
+  }
+
+  li.gsoc {
+    margin-left: 15px;
+  }
+</style>
+<div style="z-index: 20;">
+  <h3 class="black-text">Hacktoberfest 2020 and Layer5</h3>
+  Hacktoberfest is open to everyone in our global community. Whether you’re a developer, student learning to code, event host, or company of any size, you can help drive growth of open source and make positive contributions to an ever-growing community. All backgrounds and skill levels are encouraged to complete the challenge. You can find more information <a href="https://hacktoberfest.digitalocean.com/"> here </a>.
+ <ul>
+    <ol>
+      <b> Hacktoberfest is a celebration open to everyone in our global community.</b>
+    </ol>
+    <ol>
+      While small, the Layer5 community represents the largest collection of
+      service mesh projects and their maintainers in the world. We build
+      projects to provide learning environments, deployment and operational best
+      practices, performance benchmarks, create documentation, share networking
+      opportunities, and more. Our shared commitment to the open source spirit
+      pushes Layer5 projects forward. New members are always welcome.
+    </ol>
+    <ol>
+      <b> Layer5 has a couple of issues ready for hacktoberfest's participants </b>
+    </ol>
+    <ol>
+      We're preparing an interesting list of issues that will be good for this event. You can found that list <a href="https://github.com/search?q=org%3Alayer5io+label%3Ahacktoberfest&type=issues"> here </a>
```

Comment:
@jesuslerma  Can you please update the line as "You can find the list ..." ?

Reply:
Done :)